uds/translator/data_record_definitions/did.py

```
@@ -278,6 +278,96 @@ def _get_did_data(did: int) -> Tuple[RawDataRecord]:
                                         default_message_continuation=[default_did_data])
 
 
+def get_did_data_mask_2013(name: str, optional: bool) -> ConditionalFormulaDataRecord:
```

Ruff (0.14.4)

Boolean-typed positional argument in function definition

(FBT001)